appimage.go

```diff
@@ -12,7 +14,7 @@ import (
 
 func installAppImageOptionsFromCLIContext(context *cli.Context) (types.InstallOptions, error) {
 	executable := context.String("executable")
-	appName := context.Args().First()
+	context, appName := parseSourceFromAppName(context, context.Args().First())
```

Comment:
Is it possible to calculate the `appName` from the parameter too, like
```bash
zap install repo/software1
```
should automatically set the `appName = software1`

and for local files
```bash
zap install ./super-cool-x86_64.AppImage
```
it should set `appName = super-cool`